lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -241,8 +242,8 @@ def handle_event(self, event):
         return handler(event)
 
     def handle_unknown_event(self, event):
-        warnings.warn('Unhandled message type {0}. {1}'.format(
-            event['type'], event), stacklevel=2)
+        _log.warning('Unhandled message type {0}. {1}'.format(
```

Comment:
also `_warn_external`?

Reply:
This was changed to ```_log.warning``` following @anntzer's suggestion.

Reply:
In practice this will almost always get called asynchronously by the webserver event handler so "external" will point to the call to mainloop(), which is not really helpful.